### PR TITLE
adding two new folders and a template

### PR DESCRIPTION
**Pull request by bridgetegan**

### CODE REVIEW

File: video/PuppetResourceAbstration.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Video script for "Puppet code abstractions"
+
+Note: Italic font indicates alternate/extra wording
```
Comment (greglarkin) [inline video/PuppetResourceAbstration.md]: Were the italic and bold indicators stripped out of the text below? If so, these notes could be removed or replaced with a note that the `[ ... ]` phrases indicate alternate wording or an inline note/question.

Comment (greglarkin) [reply video/PuppetResourceAbstration.md]: @bridgetegan ^^^

Comment (bridgetegan) [reply video/PuppetResourceAbstration.md]: Yes, I just copied from Jira b/c I could not add an empty folder to GH. Can I fix/edit the script once the PR is merged? Or should I do that before?

Comment (greglarkin) [reply video/PuppetResourceAbstration.md]: You can do it before by making further edits in your branch locally and pushing them to GitHub. The PR here will update automatically with every new set of commits you push until it's merged into master.

**Pull request merged**